### PR TITLE
Presto stage

### PR DESCRIPTION
**Pull request by rimolive**
## This Pull Request implements

Add dh-stage-trino to run on api.rhods-idh.dev.radahub.redhat.com cluster.

## If migrating an Application to ArgoCD
- [X] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

### CODE REVIEW

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions [hunk 1]
```diff
@@ -6,6 +6,7 @@
   - https://api.thoth01.lab.upshift.rdu2.redhat.com:6443
   - https://api.thoth12.lab.upshift.rdu2.redhat.com:6443
   - https://api.ocp4.idh.dev.datahub.redhat.com:6443
+  - https://api.rhods-idh.dev.datahub.redhat.com
```
Comment (tumido) [inline manifests/overlays/prod/configs/argo_cm/resource.inclusions:9]: Looking at the other cluster URLs the port may be missing here

Comment (rimolive) [reply manifests/overlays/prod/configs/argo_cm/resource.inclusions:9]: Thanks for pointing that. Let me add the port

**Review by anishasthana — COMMENTED** [review]
/lgtm I'm good with merging this as long as we don't _actually_ include the trino app in the kustomization file at https://github.com/AICoE/aicoe-cd/blob/main/manifests/overlays/prod/applications/data_hub/kustomization.yaml

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**